src/testFixtures/java/org/opensearch/knn/KNNRestTestCase.java

```diff
@@ -48,12 +48,7 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Base64;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
There is some debate about this, but in general I like to avoid wild card imports. This is googles recommendation: https://google.github.io/styleguide/javaguide.html#s3.3-import-statements.

Reply:
ack